api/Point.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/Point",
         "support": {
           "chrome": {
-            "version_added": null
+            "version_added": false
```

Comment:
This is really `WebKitPoint`, which was in Chrome until I removed it. But it's true to say it's now not supported.

Reply:
Do you possibly know what version it was removed in?  Since we know that it was in Chrome at some point, I'd say `version_added: false` is inaccurate.  Alternatively, we can probably just do `version_added: true, version_removed: false`!